.gitlab/functional_test_junit_upload.yml

```diff
@@ -0,0 +1,23 @@
+functional_test_junit_upload:
+  # Uncomment this to only run junit uploads on main/stable branches
+  # rules:
+  #   !reference [.on_main_or_release_branch]
+  when: always
+  stage: functional_test_junit_upload
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/datadog-ci-uploader:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:main"]
+  allow_failure: true
+  needs:
+    - job: kitchen_test_system_probe_linux_x64
+      optional: true
+    - job: kitchen_test_system_probe_linux_x64_ec2
+      optional: true
+    - job: kitchen_test_system_probe_linux_arm64
+      optional: true
+  variables:
+    DD_ENV: ci
+  script:
+    - set +x
```

Comment:
Is this for debugging only? Still needed?

Reply:
Confusingly, this turns OFF logging. It is meant to hide the output from the ssm command. This is copied verbatim from `source_test_junit_upload.yml`.

Reply:
I thought it was doing the opposite :-)